buffer/src/main/java/io/netty/buffer/AbstractReferenceCountedByteBuf.java

```diff
@@ -43,7 +43,7 @@ protected long unsafeOffset() {
 
     // Value might not equal "real" reference count, all access should be via the updater
     @SuppressWarnings("unused")
-    private volatile int refCnt = updater.initialValue();
+    private final int refCnt = updater.initialValue();
```

Comment:
This does not look correct, this field is updated using the `AtomicIntegerFieldUpdater` above. You can suppress warnings using `@SuppressWarnings({"unused", "FieldMayBeFinal"})` instead

Reply:
Done